src/wanna/core/models/notebook.py

```diff
@@ -10,7 +10,6 @@
 
 class BucketMount(BaseModel, extra=Extra.forbid):
     bucket_name: str
-    bucket_dir: Path
     local_path: Path
```

Comment:
this can be confusing naming. "Local path" to what ? local to the vm or local to my workstation. 

What about simplifying it further by dropping `local_path` altogether and just mount the bucket to `/gcs/{bucket_name}` as all other VertexAI Services, inclusive managed-notebooks ?

Reply:
I changed it to mount_path, and set the default to /gcs/{bucket_name} . I think it can still be usefull to change sometimes?